hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hosted-control-planes-release-notes"]
+= Hosted control planes release notes
```

Comment:
```suggestion
= {hcp-capital} release notes
```
I think we should use the attribute here. If I'm mistaken, feel free to ignore me :)

Reply:
You are right. The attribute was not rendering so I used the term. Checking how can attribute be rendered....